.github/workflows/multi-node.yml

```diff
@@ -54,7 +54,7 @@ jobs:
             -Dakka.test.multi-node.java=${JAVA_HOME}/bin/java \
             -Dmultinode.XX:MetaspaceSize=128M \
             -Dmultinode.Xms512M \
-            -Dmultinode.Xmx1536G \
```

Comment:
1536G !?

Reply:
wow, I didn't even notice that

Reply:
That's the second largest heap I ever saw.

Reply:
ooops